release.config.js

```diff
@@ -44,10 +44,22 @@ module.exports = {
         ],
       },
     ],
+    [
+      '@semantic-release/exec',
+      {
+        prepareCmd:
+          'cd example &&  yarn install --frozen-lockfile && cd ios && pod install',
```

Comment:
nit:  theres a double space after first `&&`

I'm also wondering how you feel about putting this entire command in package.json under `scripts` and then just have this call that command by name

```js
prepareCmd: 'yarn run example-pod-install'
```

Reply:
That's a good idea. I just tried, it works fine. 
Since we have `bootstrap` script, it does a similar thing. Do you think we can merge those two together?

Reply:
Ah i see what you mean, nice catch!  So if you notice the `.yarnrc` file,  it proxies yarn to call `scripts/bootstrap.js` and if no args are passed (meaning just `yarn` is ran), it runs `yarn bootstrap`.  You stated that running `yarn` as the prepare command did not update things properly right?

Reply:
Right, only run yarn will not update the Podfile.lock.

Reply:
So in that case I don't think we can merge together, but we can use parts of it to accomplish the same thing, here is what i came up with: `"update-example-lockfiles": "yarn example install --frozen-lockfile && yarn pods"` so we don't use bootstrap but just part of it (the `yarn pods` part)